reliability/tasks/CustomizedTask.py

```diff
@@ -0,0 +1,65 @@
+from .GlobalData import global_data
+from tasks.utils.oc import oc
+import os
+import logging
+
+class OCTask():
+    def __init__(self, command, kubeconfig):
+        self.command = command
+        self.kubeconfig = kubeconfig
+        
+    def execute(self):
+        return oc(self.command.lstrip("oc "), self.kubeconfig)
+
+class FileTask():
+    def __init__(self, file, kubeconfig):
+        self.file =file
+        self.kubeconfig = kubeconfig
+        self.output = []
+        self.rc = 0
+    def execute(self):
+        with open (self.file) as f:
+            for l in f.readlines():
+                task = OCTask(l, self.kubeconfig)
+                output, rc = task.execute()
+                self.output.append(output)
+                if rc != 0:
+                    break
+        return self.output, rc
+
+class CustomizedTask():
+    def __init__(self):
+        self.output = None
+        self.code = 0
+        self.customized_task_succeeded = 0
+        self.customized_task_failed = 0
+        self.logger = logging.getLogger('reliability')
+
+    def execute_task(self, action, kubeconfig):
+        if action.startswith("oc ", 0):
+            task = OCTask(action, kubeconfig)
+        elif os.path.isfile(action):
+            task = FileTask(action, kubeconfig)
+        else:
+            self.logger.warning(f"Customized action '{action}' is not supported. Provide an oc command or a file contains lines of oc command.")
+            return None
+        outout, rc = task.execute()
+        with global_data.customized_task_lock:
+            if rc == 0:
+                self.customized_task_succeeded += 1
+                return "Customized action succeeded for action: " + action
+            else:
+                self.customized_task_failed += 1
+                return "Customized action failed for action: " + action
+
+customizedTask = CustomizedTask()
+
+
+if __name__ == "__main__":
+    kubeconfig = "/Users/qili/git/svt/reliability/kubeconfigs/kubeconfig_kubeadmin"
```

Comment:
should user be allowed to change this path? Also seems like `qili` is your private filepath, I feel like that shouldn't be in the codebase.

Reply:
@kedark3 Hi KK, thank you very much for reviewing my code! Good question. The main here is for test purpose of this single python - as we con't have UT code - should add when time is right. 'Real' run will never hit this main. But you're quite right, I'll replace the personal path to a common placeholder.

Reply:
Right, I understand that aspect of it :) But as you said, still good to replace it with a placeholder.

Reply:
https://github.com/openshift/svt/pull/676/commits/a3f6255c38c847c238463fbdf0344cc05448af61 did the change. I'll squash the commits later.

Reply:
Squashed